src/main/java/me/ramidzkh/mekae2/integration/emi/ChemicalIngredientConverter.java

```diff
@@ -26,31 +20,25 @@ public Class<?> getKeyType() {
 
     @Override
     public @Nullable EmiStack toEmiStack(GenericStack stack) {
-        try {
-            if (stack.what() instanceof MekanismKey key) {
-                return JemiUtil.getStack(key.withAmount(Math.max(1, stack.amount())));
-            }
-        } catch (Exception e) { // catch error in case JEMI internals change
-            LOGGER.error("Failed to convert GenericStack to EmiStack", e);
+        if (stack.what() instanceof MekanismKey key) {
+            return IMekanismAccess.INSTANCE.emiHelper().createEmiStack(key.getStack().getType(), stack.amount());
```

Comment:
Does this nuke potential NBT?

Reply:
Currently ChemicalStack has no NBT, only the type and amount